testgrid/config/config.yaml

```diff
@@ -3731,15 +3731,15 @@ dashboards:
   - name: kubeadm-gce-1.6-on-1.7
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7
   - name: kubeadm-gce-upgrade-1.6-1.7
```

Comment:
Would be nice to have periodic in the name so I can tell which tests are periodic and which are ci in the grid

Reply:
I don't want to make the names too long.
It's one click away on a green or red tile if you want to know that any way...